pkg/agent/util/net_windows.go

```diff
@@ -1265,13 +1265,19 @@ func getAdaptersByName(name string) ([]adapter, error) {
 // flags to return addresses in all routing compartments.
 const GAA_FLAG_INCLUDE_ALL_COMPARTMENTS = 0x00000200
 
+// GAA_FLAG_INCLUDE_ALL_INTERFACES is used in windows.GetAdapterAddresses parameter
+// flags to return addresses for all NDIS interfaces.
+const GAA_FLAG_INCLUDE_ALL_INTERFACES = 0x00000100
+
 // adapterAddresses returns a list of IpAdapterAddresses structures. The structure
 // contains an IP adapter and flattened multiple IP addresses including unicast, anycast
 // and multicast addresses.
 // This function is copied from go/src/net/interface_windows.go, with a change that flag
-// GAA_FLAG_INCLUDE_ALL_COMPARTMENTS is introduced to query interfaces in all compartments.
+// GAA_FLAG_INCLUDE_ALL_COMPARTMENTS is introduced to query interfaces in all compartments,
+// and GAA_FLAG_INCLUDE_ALL_INTERFACES is introduced to query all NDIS interfaces even they
+// are not configured with any IP addresses, e.g., uplink.
```

Comment:
remove "," before uplink

Reply:
According to this https://jakubmarian.com/comma-after-i-e-and-e-g/, a commar after "e.g." is valid
```
Virtually all American style guides recommend to follow both “i.e.” and “e.g.” with a comma (just like if “that is” and “for example” were used instead), so the very same sentence in American English would become:
```

Reply:
Got it.